packages/next/src/server/base-server.ts

```diff
@@ -567,7 +581,7 @@ export default abstract class Server<ServerOptions extends Options = Options> {
   private handleRSCRequest: RouteHandler = (req, _res, parsedUrl) => {
     if (!parsedUrl.pathname) return false
 
-    if (this.normalizers.prefetchRSC.match(parsedUrl.pathname)) {
+    if (this.normalizers.prefetchRSC?.match(parsedUrl.pathname)) {
```

Comment:
Are these going to have a similar issue as we saw with `urlPathname` needing to be used because `basePath` isn't stripped from `parsedUrl.pathname`?

Reply:
For all the `rsc` requests, we don't have a prefix, it only affects the suffix. I agree that we need a more comprehensive solution for pathname management here, but essentially that's just re-introducing a router here 😅